CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to Lion
+## Branch names
+Make sure that your branch names follow our naming scheme `author/feature_name`
```

Comment:
I still like dev/author/thing

Reply:
i like `author--thing` :)

Reply:
We've been doing author/thing for the most part so we should prob stick to that
Im not against `dev/author/thing`, but is the `dev` not redundant? Is there ever a time someone shouldn't put `dev`?

Reply:
I think something that hasn't been brought up is the situation where branches have multiple authors. A lot of times people will ask for help after they create a branch, and it seems a bit unfair for it to be labeled only under one person.

Reply:
I agree with this, what a good naming convention for when it's multiple people?
Putting each name is too long

Reply:
I'd say maybe rely on the participants/assignee list on github or listing the authors in the pull request? Still probably not the best way though

Reply:
Expanding on what Elbert said not only are people assigned for the ticket they want to work on but the assigned list is dynamic. Branch names are not dynamic, you have to manually check another one out in order to name it appropriately, and I think everyone can agree that branch names with multiple authors isn't the nicest way to go about doing things. Luckily git history fills this in for us users are automatically audited and tagged in PR based on the git history, in other words, it's dynamic, there's no question of who did what. I just feel like keeping author names in the branch name causes more issues than they solve.

Reply:
I'd agree with this. I'm a fan of `category/specific`, with as many levels of depth as you need.

For an example, if you were fixing a bug in the AI side of Connect Four, you'd open the branch `bugfix/ai/connect_four`, this leaves no ambiguity, and allows got'a branch name autocomplete to do its job nicely in the CLI.

Got already has capabilities to manage authors in both commits and PRs.

Reply:
If we want to stick with `author/thing` then we would need a way to make sure that the first person that assigns themselves to the issue is the one who makes the branch. I do like the idea of `category/specific` though. We would just need a list of categories which would may or may not be all encompassing and can maybe correlate to the tags we have for issues.